vsintegration/src/FSharp.Editor/Common/Pervasive.fs

```diff
@@ -199,13 +199,4 @@ module Async =
         async { return! agent.PostAndAsyncReply id }
         
 
-type AsyncBuilder with
```

Comment:
I took these out because it is important to be able to search the codebase for all **explicit** conversions from Task --> Async, and check that a cancellation token is being passed in for each one